plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialLevel.java

```diff
@@ -90,118 +130,34 @@ public static void register() {
         // This will return 0 for all valid materials aside from snow.
         // -->
         PropertyParser.registerStaticTag(MaterialLevel.class, ElementTag.class, "minimum_level", (attribute, material) -> {
-            return new ElementTag(material.getMin());
+            return new ElementTag(material.getBlockData() instanceof Snow snow ? snow.getMinimumLayers() : 0);

```

Comment:
This is simple logic, but it's still being repeated here, I'd keep it in a method as this property could definitely support more stuff in the future that have a non-0 min value.